modules/standard/Time.chpl

```diff
@@ -107,12 +107,19 @@ module Time {
       controlled with :var:`cIsoDayOfWeek`.
    */
   enum dayOfWeek {
+    /**/
     Monday = firstDayOfWeekNum,
+    /**/
     Tuesday,
+    /**/
     Wednesday,
+    /**/
     Thursday,
+    /**/
     Friday,
+    /**/
     Saturday,
+    /**/
```

Comment:
Hrm.  This seems unfortunate - the names are pretty self-evident, so I agree that they don't need anything, but we also don't want them to constantly trigger if we're going to have the script run as something like a Github Action

Reply:
I agree, I considered something more heavy handed like `@chpldoc.ignoreUndocumented` applied to the `enum` (psuedo-name) but I think this approach makes more sense. This doesn't change how chpldoc renders the enum elements, but it does make the code a bit uglier